packages/webapp/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import ReactTable from './index';
+import MuiTable from './v2';
```

Comment:
I think we should import these as `TableV1` and `TableV2` -- the implementation details of which library we used to build them should be transparent to the rest of the components. It would also be nice for clarity to move the files related to v1 to their own v1 folder, similarly to v2.

Reply:
I can rename.. I thought it might be a whole thing to edit the table structures. 

In my opinion I would like to change `Table/index.jsx` to `Table/TableV1.jsx` and `Table/Tables.tsx` to `Table/index.tsx` and move `Table/v2/index` and `Table/v2/styles` out of v2 folder to top level. With Headers getting its own folder too for Table/Headers/TableHead.jsx.